connect/src/com/telenor/connect/id/IdToken.java

```diff
@@ -23,7 +34,16 @@ public IdToken(String serializedSignedJwt) {
                     "Could not parse saved id token. idToken=" + this, e);
         }
 
+        final Map<String, Object> cc = jwtClaimsSet.getCustomClaims();
```

Comment:
Moved it down. I think I'll use to option to squash the commits for this branch when I merge.